apps/L1G/tinder/tinder-frontend/src/components/ProfileImages.tsx

```
@@ -13,15 +13,19 @@ export const ProfileImages = () => {
   const cloudName = process.env.NEXT_PUBLIC_CLOUDINARY_CLOUD_NAME;
   const fileInputRef = useRef<HTMLInputElement | null>(null);
 
+  const [uploadImagesMutation, { data, loading: mutationLoading, error: mutationError }] = useUploadImagesMutation();
```

ESLint

[error] 'data' is assigned a value but never used.

(no-unused-vars)